03-attribute-operations.Rmd

```diff
@@ -275,15 +255,15 @@ world_continents = world %>%
 world_continents
 ```
 
-`sf` objects are well-integrated with the **tidyverse**, as illustrated by the fact that the aggregated objects preserve the geometry of the original `world` object.
+`sf` objects are well-integrated with the **tidyverse**, as illustrated by the fact that the aggregated objects preserve the geometry of the original `world` object.^[Such a spatial aggregation of polygon data is know as "to dissolve polygons" in the GIS world.]
```

Comment:
Any idea why some borders are still preserved? The same happens if using a GIS in the background:

    test <- run_qgis("saga:polygondissolvebyattribute", POLYGONS = world, FIELD_1 = "continent",
                      DISSOLVED = "out.shp", BND_KEEP = "False", load_output = TRUE)

Hence, I guess the input geometry is somewhat unclean...